services/core-web/src/tests/components/Forms/Securities/BondForm.spec.js

```diff
@@ -17,6 +17,8 @@ const setupProps = () => {
   props.submitting = false;
   props.provinceOptions = MOCK.DROPDOWN_PROVINCE_OPTIONS;
   props.bondTypeOptions = [];
+  props.bondStatusOptionsHash = MOCK.BULK_STATIC_CONTENT_RESPONSE.bondStatusOptions;
+  // eslint-disable-next-line prefer-destructuring
```

Comment:
Is there a reason to not destructure here? If you disable a linting rule, please put a comment as to why.

Reply:
I googled it briefly to figure out what it was complaining about but I couldn't figure it out exactly. I didn't even add the line it's complaining about so I'm not entirely why it started to complain about it.

Reply:
Can you please add a comment as such?

Reply:
Apparently this is the fix for it. So silly
`
[props.bond] = MOCK.BONDS.records[0];
`